Gemfile

```diff
@@ -1,11 +1,13 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf',  '~> 3.2'
-gem 'chefspec',   '~> 4.2'
-gem 'foodcritic', '~> 4.0'
-gem 'rubocop',    '~> 0.30.1'
+gem 'berkshelf', '~> 4.3.5'
```

Comment:
These are some pretty rough pins here.  I'd stick to feature so we can float other than major breaking changes.  Since they're just testing deps that's a safe thing to do